R/file_cache.R

```diff
@@ -0,0 +1,542 @@
+#' Create a file cache object
+#'
+#' This creates a file cache which is to be used by sass for caching generated
+#' .css files.
+#'
+#' @param dir The directory in which to store the cached files. If `NULL` (the
+#'   default), then it will be set to a subdirectory of the system temp
+#'   directory named `R-sass-cache-username`. Because this cache directory is
+#'   not in the R process's temp directory, it will persist longer than the R
+#'   process, typically until a system reboot. Additionally, it will be shared
+#'   across R processes for the same user on the same system.
+#' @param max_size The maximum size of the cache. If the cache grows past this
+#'   size, the least-recently-used objects will be removed until it fits within
+#'   this size.
+#'
+#' @seealso sass_get_default_cache
+#'
+#' @examples
+#' \dontrun{
+#' # Create a cache with the default settings
+#' cache <- sass_file_cache()
+#'
+#' # Clear the cache
+#' cache$reset()
+#' }
+#'
+#' @export
```

Comment:
Wouldn't it make sense to document this on the same page as the other caching functions?

Reply:
If we do keep this on a separate page, let's add a `@return` section that points to `FileCache`

Reply:
I think it makes sense to have this separate, because it's information overload for most people. For those who are really interested in the details, they can follow links from `sass_file_cache()` (which I've added) to get here.